examples/play_buffer.py

```diff
@@ -3,6 +3,28 @@
 # Author:   A.Invernizzi (@albestro on GitHub)
 # Date:     Jun 03, 2020
 
+# MIT License <http://OpenSource.org/licenses/MIT>
+#
+# Copyright (C) 2017-2018 -- mrJean1 at Gmail dot com
```

Comment:
One last thing: could you change the copyright line to
    # Copyright (c) 2020 -- A. Invernizzi, @albestro on github
or
    # Copyright (c) 2020 -- the VideoLAN team
for a copyright assignment to the videolan project.

Reply:
I'm sorry...completely missed it, just copy&paste. My fault.